dandi/utils.py

```diff
@@ -716,7 +716,8 @@ def is_url(s):
 
     TODO: redo
     """
-    return s.lower().startswith(("http://", "https://", "dandi://"))
+    return s.lower().startswith(("http://", "https://", "dandi:"))
+    # Slashes are not required after "dandi:" so as to support "DANDI:<id>"
```

Comment:
Indeed, Wikipedia says

```
URI = scheme:[//authority]path[?query][#fragment]
```

Reply:
Just then spotted it says URI not URL, so remains unclear but https://url.spec.whatwg.org/ suggests that could be used interchangeably... So whatever, let it be this way